packages/typespec-ts/test/integration/azureClientGeneratorCoreAccess.spec.ts

```diff
@@ -21,9 +21,9 @@ describe("Access Client", () => {
         .path(
           "/azure/client-generator-core/access/publicOperation/noDecoratorInPublic"
         )
-        .get({ queryParameters: { name: "myname" } });
+        .get({ queryParameters: { name: "sample" } });
       assert.strictEqual(result.status, "200");
-      assert.strictEqual(result.body.name, "myname");
+      assert.strictEqual(result.body.name, "sample");
```

Comment:
Refer to https://github.com/Azure/cadl-ranch/pull/755/files#diff-c34d87bd2aa1d671fa50bd3c4dfe5fbb77e6d8b78cccb65a2e1bc4359ae587d8